### DISCUSSION TITLE
Without zwift

### DISCUSSION THREAD

**Discussion by posssson** [doudar/SmartSpin2k#338] [Ideas]
Hi everone,

I found this https://github.com/dvmarinoff/Flux to control the hometrainer smartspin with a web browser and it is work perfecty and better than zwift and free :) Have fun

**Comment by doudar** [comment]
That's a pretty cool project!

Thanks!

**Comment by eMadman** [comment]
I'm playing with it off of my computer and an SS2K on my desk right now.  Got it working with no effort.  It's pretty cool!

![image](https://user-images.githubusercontent.com/24726844/161175753-60cca9ad-7688-4a36-9c0c-3b677b9b322d.png)
